pkg/policy/distillery_test.go

```diff
@@ -650,11 +641,8 @@ func Test_MergeL3(t *testing.T) {
 				if err != nil {
 					t.Errorf("Policy resolution failure: %s", err)
 				}
-				if equal, err := checker.ExportedEqual(mapstate, tt.result); !equal {
-					t.Logf("Rules:\n%s\n\n", api.Rules(rules).String())
-					t.Logf("Policy Trace: \n%s\n", logBuffer.String())
-					t.Errorf("Policy obtained didn't match expected for endpoint %s:\n%s\nObtained: %v\nExpected: %v", labelsFoo, err, mapstate, tt.result)
-				}
+				require.EqualExportedValuesf(t, tt.result, mapstate,
```

Comment:
Could we use `if equal := assert.EqualExportedValuesf(...); !equal {` here and still keep the log messages?

(same for the ones below)

Reply:
thanks, personally I didn't think the log is useful, however, let me revert it back just in case others are replying on existing logs.